libdevice/cmake/modules/SYCLLibdevice.cmake

```diff
@@ -32,6 +32,15 @@ set(compile_opts
   -sycl-std=2020
   )
 
+if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
```

Comment:
What was the reason for this change @aelovikov-intel ?

Reply:
https://github.com/intel/llvm/pull/7761/

Reply:
Our testing on a system with older libc where gcc installation is in non-default location (as default is too old).